bin/deploy-hobby

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -e
+
+# Seed a secret
+POSTHOG_SECRET=$(echo $RANDOM | md5sum | head -c 25)
+export POSTHOG_SECRET
+
+# Talk to the user
+echo "Welcome to the single instance PostHog installer 🦔"
+echo ""
+echo "⚠️  You really need 4gb or more of memory to run this stack ⚠️"
+echo ""
+echo "Let's first start by getting the exact domain PostHog will be installed on"
+echo "Make sure that you have a Host A DNS record pointing to this instance!"
+echo "This will be used for TLS 🔐"
+echo "ie: test.posthog.net"
+read -r DOMAIN
+export DOMAIN=$DOMAIN
+echo "Ok we'll set up certs for https://$DOMAIN"
+echo ""
+echo "Do you have a Sentry DSN you would like for debugging should something go wrong?"
+echo "If you do enter it now, otherwise just hit enter to continue"
+read -r SENTRY_DSN
+export SENTRY_DSN=$SENTRY_DSN
+echo ""
+echo "We will need sudo access so the next question is for you to give us superuser access"
+echo "Please enter your sudo password now:"
+sudo echo ""
+echo "Thanks! 🙏"
+echo ""
+echo "Ok! We'll take it from here 🚀"
+
+echo "Making sure any stack that might exist is stopped"
+sudo -E docker-compose -f docker-compose.hobby.yml stop || true
+
+# update apt cache
+echo "Grabbing latest apt caches"
+sudo apt update
+
+# clone posthog
+echo "Installing PostHog 🦔 from Github"
+sudo apt install -y git
+rm -rf posthog
+git clone https://github.com/PostHog/posthog.git 
+
+# rewrite caddyfile
+rm -f Caddyfile
+envsubst > Caddyfile <<EOF
+$DOMAIN, :80, :443 {
+reverse_proxy http://web:8000
+}
+EOF
+
+# write entrypoint
+rm -rf compose
+mkdir -p compose
+cat > compose/start <<EOF
```

Comment:
instead of rewrite another start script, can we reuse something we already have?